### PR TITLE
Add cosmiand SELinux module on RHEL

### PR DESCRIPTION
**Pull request by grydz**

### CODE REVIEW

File: .github/workflows/build_all.yml [hunk 1]
```diff
@@ -25,15 +25,12 @@ jobs:
...
-          python3 -m pip install ansible-lint
-
       - name: Run ansible-lint
-        run: ansible-lint
+        uses: ansible/ansible-lint@main
+        with:
+          args: ""
+          setup_python: "true"
+          working_directory: "./ansible"
+          requirements_file: "requirements.yml"
```
Comment (Manuthor) [inline .github/workflows/build_all.yml:36]:
```suggestion
        uses: ansible/ansible-lint@main
        with:
          setup_python: true
          working_directory: ansible
          requirements_file: requirements.yml
```

File: ansible/roles/selinux/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  block:
+    - name: Install policycoreutils
+      ansible.builtin.dnf:
+        name: policycoreutils
+        state: present
+        update_cache: true
+
+    - name: Download SELinux module from package.cosmian.com
+      ansible.builtin.get_url:
+        url: https://package.cosmian.com/selinux-module/last_build/new-custom-label/cosmiand.pp
```
Comment (Manuthor) [inline ansible/roles/selinux/tasks/main.yml:15]: just let a comment here to remember to tag `cosmiand.pp` before merging this PR

**Review by Manuthor — APPROVED** [review]

File: .github/workflows/build_all.yml (hunk 1 above)
Comment (grydz) [reply .github/workflows/build_all.yml]: Done in 791141166d39e7d5ad3b9d91ba96f80940885457.

File: ansible/roles/selinux/tasks/main.yml (hunk 2 above)
Comment (grydz) [reply ansible/roles/selinux/tasks/main.yml]: Done in 791141166d39e7d5ad3b9d91ba96f80940885457.

**Pull request merged**